.github/workflows/codecov.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: codecov
```

Comment:
@advasileva Can we move GA into a separate PR, please?

Reply:
@volodya-lombrozo Done in #41

Reply:
@advasileva What do you think if we remove that code from the current PR since it's already in other PR?